dipy/utils/tripwire.py

```diff
@@ -35,7 +36,8 @@ class TripWire(object):
     ...     import silly_module_name
     ... except ImportError:
     ...    silly_module_name = TripWire('We do not have silly_module_name')
-    >>> silly_module_name.do_silly_thing('with silly string') #doctest: +IGNORE_EXCEPTION_DETAIL
+    >>> silly_module_name.do_silly_thing('with silly string') \
+    #doctest: +IGNORE_EXCEPTION_DETAIL
```

Comment:
Please put that one back on the same line as it was. This line-break is not needed.